assets/css/style.css

```diff
@@ -0,0 +1,198 @@
+*{
+    padding: 0;
+    margin: 0;
+    top: 0;
+    box-sizing: border-box;
+}
+.container{
+    max-width:1440px;
+    margin: 0 auto;
+}
+#my{
+    height: 540px;
+    display: flex;
+    flex-direction: column;
+    background-image: url(https://web.archive.org/web/20140301004302im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg);
```

Comment:
Your background image is little off from the right position. In order to fix that, please replace the `background-image` with this one: `background: url(https://web.archive.org/web/20140301004302im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg) 0 100%;`